learn/advanced/filtering_and_faceted_search.md

```diff
@@ -77,61 +73,189 @@ A basic condition could request movies containing the `horror` genre:
+
+The following expression returns all action movies:
+
+```
+genres = action
+```
+
+::: note
+The equality operator does not return any results for `null` and empty arrays.
 :::
 
+#### Inequality
+
+The inequality operator (`!=`) returns all documents not selected by the equality operator. When operating on strings, `!=` is case-insensitive.
+
+The following expression returns all movies without the `action` genre:
+
+```
+genres != action
+```
+
+#### Comparison
+
+The comparison operators (`>`, `<`, `>=`, `<=`, `TO`) select documents satisfying a comparison. Comparison operators only apply only to numerical values.
+
+The expression below returns all documents with a user rating above 85:
+
+```
+rating.users > 85
+```
+
+To filter documents with a user rating of 80 or above but below 90, you would use:
+
+```
+rating.users >= 80 AND rating.users < 90
+```
+
+##### `TO`
+
+`TO` is equivalent to `>= AND <=`. The following expression returns all movies with a user rating of 80 or above but below 90:
+
+```
+rating.users 80 TO rating.users 89
+```
+
+#### `EXISTS`
+
+The `EXISTS` operator checks for the existence of a field. Fields with empty or `null` values count as existing.
+
+The following expression returns all documents that contain the `release_date` field, even if it is empty or `null`:
+
+```
+release_date EXISTS
+```
+
+The negated form of the above expression can be written as:
+
+```
+release_date NOT EXISTS
+NOT release_date EXISTS
+```
+
+Both forms are equivalent.
+
+#### `IN`
+
+`IN` combines equality operators by taking an array of comma-separated values delimited by square brackets. It selects all documents whose chosen field contains at least one of the specified values.
+
+Both of the following expressions are equivalent and return all documents whose `genres` includes either `horror`, `comedy`, or both:
+
+```
+genres IN [horror, comedy]
+genres = horror OR genres = comedy
+```
+
+The negated form of the above expression can be written as:
+
+```
+genres NOT IN [horror, comedy]
+NOT genres IN [horror, comedy]
+```
+
+Both are equivalent and mean:
+
+```
+genres != horror AND genres != comedy
+```
+
+#### `NOT`
+
+The negation operator (`NOT`) selects all documents that do not satisfy a condition. It has higher precedence than `AND` and `OR`.
+
+The following expression will return all documents whose `genres` does not contain `horror` and documents with a missing `genres` field:
+
+```
+NOT genres = horror
+```
+
 ### Filter expressions
 
-You can build filter expressions by grouping basic conditions. Filter expressions can be written as strings, arrays, or a mix of both.
+You can build filter expressions by grouping basic conditions using `AND` and `OR`. Filter expressions can be written as strings, arrays, or a mix of both.
 
 ::: warning
 The [`GET` route of the search endpoint](/reference/api/search.md#search-in-an-index-with-get-route) only accepts string filter expressions.
 :::
```

Comment:
If there's a warning about this in the GET search endpoint API reference, then I think we can remove this. It's not specific to filters, it's because GET requests don't accept payloads so everything has to be submitted as a parameter in the route. Also, GET is not very used so all in all, I think it's not worth derailing the topic of the guide.